routes/index.js

```diff
@@ -257,10 +257,12 @@ module.exports = function () {
 
   async function showDoc (req, res) {
     if (req.params[0]) {
-      const page = req.params[0]
-      const BASE = 'https://raw.githubusercontent.com/datahq/content/master/'
+      const page = req.params[0] 
+      const path = req.params[0] ? 'docs/' + req.params[0] + '.md' : 'README.md'
```

Comment:
It just doesn't make sense since you're already inside if statement and can be sure there is a `req.params[0]`. This line is really redundant - just remove it.